apps/src/labs/projects/ProjectManager.ts

```diff
@@ -339,4 +382,12 @@ export default class ProjectManager {
       error: error.toString(),
     });
   }
+
+  // Helpers
+  /**
+   * This method exists to be mocked for unit tests.
+   */
+  getLocation() {
```

Comment:
Should we just use `getLocation()` from apps/src/utils here? Looks like returns `window.location` but should work the same?